src/user/Activity/Users.js

```diff
@@ -81,7 +81,7 @@ class Users extends Component {
               showSizeChanger
               onShowSizeChange={this.onShowSizeChange}
               defaultCurrent={1}
-              total={500}
+              total={users.length}
```

Comment:
set default value too

Reply:
default as in?

Reply:
like if users are undefined then it should not break this component, so assign a default value as well.
total={users.length || 10}

Reply:
Got you! I'll make this change asap!

Reply:
Pushed the changes!